source/appModuleHandler.py

```diff
@@ -109,6 +109,9 @@ def getAppModuleFromProcessID(processID):
 		mod=runningTable.get(processID)
 		if not mod:
 			appName=getAppNameFromProcessID(processID)
+			# #5323: Certain executables contain dots as part of its file name.
+			if "." in appName:

```

Comment:
I'd remove this if check. The reason is that replace has to search the string anyway and it will be a no-op if nothing is replaced, so there's no point in searching it twice.